Diagnostic/services/metrics-extension.service

```diff
@@ -3,7 +3,7 @@ Description=Metrics Extension service for Linux Agent metrics sourcing
 After=network.target
 
 [Service]
-ExecStart=%ME_BIN% -TokenSource MSI -Input influxdb_udp -InfluxDbUdpPort %ME_INFLUX_PORT% -DataDirectory %ME_DATA_DIRECTORY% -LocalControlChannel -MonitoringAccount %ME_MONITORING_ACCOUNT% -LogLevel Error
```

Comment:
What was the error happening before the fix ? Do we have quick tsg for the particular error to look for when investigating that is being resolved with this change? since this fix will make to only newer versions of AMA extension.

Reply:
Updated the description -- this PR is not addressing any AMA functionality issues on CIS-hardened images (none have been found). Rather, it is making changes to ensure that AMA installation does not cause the system to become out of compliance with the CIS benchmark. In this case, ME was listening on all interfaces, because if you don't specify the host, it will listen on 0.0.0.0. With this change now we ensure ME is only opening a port on localhost interface.

Reply:
Thanks Henry for details. Signed off.